ciao-controller/openstack_compute.go

```diff
@@ -74,6 +77,234 @@ func instanceToServer(ctl *controller, instance *types.Instance) (compute.Server
 	return server, nil
 }
 
+func validateBlockDeviceMappingSourceType(srcType string) (err error) {
+	validSourceTypes := []string{
```

Comment:
We're conflicted stylistically through out the code on this topic.  I tend to leave them unnamed, I think?  But I also tend to copy/paste.

I've also spotted some code that does stuff like a dozen instances of exactly "return varA, varB, err", with the var's being inconsistently set/unset and err being something or nil.  I like an explicit "return nil, nil, err" or "return varA, varB, nil" distinguishing quickly visually success and error paths, but we don't consistently do that either.

I'm somewhat surprised we don't get dinged by 'go vet' on this.

@markdryan do you have any idiomatic guidance in this area?